src/lib/zork/index.ts

```diff
@@ -287,7 +349,7 @@ const onSocksConnection = (ctx: Context, sessionId: any) => {
       return this;
     },
     handleDisconnect: () => {
-      channel.close();
+      //channel.close();  // TODO need this?
```

Comment:
@trevj I originally commented this out because it didn't seem right to close datachannels from the giver side, especially if we want to support legacy clients which recycle datachannels in the pool.

But ALSO, if I uncomment this out, then at some point Zork inevitably crashes while proxying with `'node build/src/lib/zork/index.j…' terminated by signal SIGSEGV (Address boundary error)` errors! Should I replace this commented out line with a comment like "we don't close the datachannel here because..."?